cmd/configui/index.html

```diff
@@ -222,15 +222,19 @@ <h5 class="modal-title" id="exampleModalLabel">Add Splunk HEC </h5>
             var reader = new FileReader();
             reader.readAsText(csvFile, "UTF-8");
             reader.onload = function(e){
-                var rows = e.target.result.split("\n");
+                var rows = e.target.result.trim().split("\n");
                 for (var i = 0; i < rows.length; i++) {
                     console.log('row', i)
-                    var cells = rows[i].split(",");
+                    var cells = rows[i].trim().split(",");
                     if (cells.length==3){
+                        isValidfieldsFlag = true
+
                         //  Check if each row in the CSV file has 3 values
-                        if (!isValidIP(cells[0])){

```

Comment:
Lines 231 to 238 can be written in one line. 
isValidIPFlag = isValidIP(cells[0])